.github/workflows/yaml-lint.yaml

```diff
@@ -0,0 +1,18 @@
+name: Yamllint Github Actions
+on: 
+  pull_request:
+    branches: 
+      - master
+
+jobs: 
+  lintFalcoRules: 
+    name: Yamllint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3 
+      - name: checkout falco-rules
+        uses: ibiqlik/action-yamllint@v3
+        with: 
+          files: /rules/*.yaml 
```

Comment:
Sorry for that @FedeDP , we should use `file_or_dir: rules/*.yaml` . Let me change it. Should we change it to 
```
steps:
  - uses: actions/checkout@v4
  - name: Checkout repo
    uses: ibiqlik/action-yamllint@v3
    with: 
      file_or_dir: rules/*.yaml
```
WDYT?🤔